src/test/java/uk/gov/hmcts/reform/ccd/config/TestConfig.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.ccd.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.ccd.service.remote.clients.DocumentClient;
+
+import static org.mockito.Mockito.mock;
+
+@Configuration
+public class TestConfig {
+
+    @Bean
+    public DocumentClient documentClient() {
+        return mock(DocumentClient.class);
```

Comment:
I don't understand why DocumentClient needs to be mocked but not the other FeignClients. There must be some misconfiguration for DocumentClient that does not affect the other clients.

I believe we need to completely remove TestConfig and fix the issue with DocumentClient

Reply:
Test was looking for any OpenFeignClient bean definition not particularly DocumentClient feignClient this has been resolved using @ImportAutoConfiguration({FeignAutoConfiguration.class})